From cdf083c52e19bb6504aed0e13ee59dd16b02a1a1 Mon Sep 17 00:00:00 2001
From: Christophe Roullier <christophe.roullier@foss.st.com>
Date: Tue, 12 Mar 2024 09:32:07 +0100
Subject: [PATCH 1022/1141] dt-bindings: net: snps,dwmac: add snps,ext-systime
 property

Used to allow ptp external system time

Change-Id: I5eb8d850c6cab01faa893ee98e5cb3ef80f9e1dd
Signed-off-by: Gwenael Treuveur <gwenael.treuveur@st.com>
Signed-off-by: Christophe Roullier <christophe.roullier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/294638
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/366026
---
 Documentation/devicetree/bindings/net/snps,dwmac.yaml | 5 +++++
 1 file changed, 5 insertions(+)

--- a/Documentation/devicetree/bindings/net/snps,dwmac.yaml
+++ b/Documentation/devicetree/bindings/net/snps,dwmac.yaml
@@ -294,6 +294,11 @@ properties:
     description:
       Frequency division factor for MDC clock.
 
+  snps,ext-systime:
+    $ref: /schemas/types.yaml#/definitions/flag
+    description:
+      Enable PTP External System Time bus.
+
   mdio:
     $ref: mdio.yaml#
     unevaluatedProperties: false
